recipes/virtual_dataframe/conda-forge.yml

```diff
@@ -0,0 +1,11 @@
+channels:
+  # sources selects the channels to pull packages from, in order.
+  sources:
+    - conda-forge
+    - rapidsai
```

Comment:
just a heads-up: this probably won't get approved

Reply:
It's not possible to have a package with a dependency from another channel ?